src/main/java/at/hannibal2/skyhanni/features/garden/GardenPlotAPI.kt

```diff
@@ -97,7 +101,7 @@ object GardenPlotAPI {
         getData()?.apply { sprayHasNotified = true }
     }
 
-    private fun Plot.setSpray(spray: SprayType, duration: Duration) {
+    private fun Plot.setSpray(spray: SprayType?, duration: Duration) {
```

Comment:
instead of nullable, create a new function `removeSpray` that doesnt have duration as parameter